integ-test/build.gradle

```diff
@@ -101,11 +106,36 @@ testClusters.all {
 
 testClusters.integTest {
     plugin ":opensearch-sql-plugin"
+    keystore 'plugins.query.federation.catalog.config', new File("$projectDir/src/test/resources/catalog/", 'catalog.json')
+}
+task startPrometheus(type: SpawnProcessTask) {
+    doFirst {
+        download.run {
+            src 'https://github.com/prometheus/prometheus/releases/download/v2.39.1/prometheus-2.39.1.linux-amd64.tar.gz'
+            dest new File("$projectDir/bin", 'prometheus.tar.gz')
+        }
+        copy {
+            from tarTree("$projectDir/bin/prometheus.tar.gz")
+            into "$projectDir/bin"
+        }
+        copy {
+            from "$projectDir/bin/prometheus.yml"
+            into "$projectDir/bin/prometheus-2.39.1.linux-amd64/prometheus"
```

Comment:
minor issue, maybe rename `prometheus-<version>.<platform>-<arch>` to `prometheus` so the directory name is fixed? or `bin/prometheus-*`, not sure if `*` is supported

Reply:
tar file gets unzipped into prometheus-2.39.1.linux-amd64. Need to introduce an unnecessary copy step in between to prometheus.  

`prometheus-2.39.1.linux-amd64` will remain for sometime. I don't have plans to change this in near term. If one changes the tar link, he can make change to folder name as well.

Reply:
seems there's no easy way to do `--strip-component` in gradle.. this needs to change when other platforms support is added, i think it's ok to change then, maybe use a constant